app/main.py

```diff
@@ -100,10 +101,14 @@ def grab_secrets(self):
                    output_path = os.path.join(self.secrets_output_folder, key_name)
                    if secret.kid is not None:
                          _logger.info('Secret is backing certificate. Dumping private key and certificate.')
-                         self.dump_pfx(secret.value, key_name)
+                         if secret.content_type == 'application/x-pkcs12':
+                               self.dump_pfx(secret.value, key_name)
+                         else:
+                               _logger.error('Secret is not in pkcs12 format')
+                               sys.exit(1)
```

Comment:
I believe you should have imported `sys`

Reply:
fixed